src/client.js

```diff
@@ -160,7 +174,8 @@ export default class StitchClient {
         useRefreshToken: true,
         rootURL: this.rootURLsByAPIVersion[v2][API_TYPE_CLIENT]
       }
-    ).then(() => this.auth.clear()); // eslint-disable-line space-before-function-paren
+    ).then(() => this.auth.clear())
```

Comment:
[minor] I think we need a `catch` or a second arg to `then` such that in both cases we succeed and always return with `this.auth.clear`. We can log the failure.

Reply:
Good call